.env.example

```diff
@@ -0,0 +1,4 @@
+# Set these in .env.development
```

Comment:
# Gatsby Auth0 Setup

<!-- 
    This is just a sample intro. Replace with whatever makes more sense to your tutorial! -->

Welcome! This PR acts as a tutorial to walk you through setting up {[Auth0](https://auth0.com)}, a service which allows for serverless login flows. It was inspired by [this gatsby livestream](https://www.gatsbyjs.org/blog/2019-03-21-add-auth0-to-gatsby-livestream). as well as the [store.gatsbyjs.org approach to authentication](https://github.com/gatsbyjs/store.gatsbyjs.org/blob/master/src/utils/auth.js).

<!-- 
    Link this one to the beginning of the code range of the first 
    comment. -->

### [START WALKTHROUGH →](https://github.com/your-org/your-repo/pull/1/files#diff-abcdef1234567890)

<!-- 
    Same goes for these. Always link to the beginning of the topic's 
    code range. -->

### Index
* {[Topic 1](https://github.com/your-org/your-repo/pull/1/files#diff-abcdef1234567890)}
* {[Topic 2](https://github.com/your-org/your-repo/pull/1/files#diff-abcdef1234567890)}
* {[Topic 3](https://github.com/your-org/your-repo/pull/1/files#diff-abcdef1234567890)}
* {[Topic 4](https://github.com/your-org/your-repo/pull/1/files#diff-abcdef1234567890)}